### DISCUSSION TITLE
workshops ... versions of Windows

### DISCUSSION THREAD

**Discussion by martin12333** [edrx/eev#8] [Ideas]
I am reading 

http://anggtwu.net/2021-2022-workshops.html

I wonder: people in your workshops ... which versions of Windows they have ?

// (C:\Windows\System32\curl.exe is preinstalled in win 10)
// (powershell 5 preinstalled in win 10) 

// powershell 2 preinstalled in win 7

EDIT: the only statistics i could find:

https://gs.statcounter.com/windows-version-market-share/desktop/brazil

https://gs.statcounter.com/windows-version-market-share/desktop/brazil/2020  // some change in browsers at the end of 2020 made it difficult to collect some sources of statistics  https://www.netmarketshare.com/

 // git-for-Windows, with bash and 100 utils and tclsh , is available even for Windows xp. (btw, my dad had wxp till 2017)

**Comment by edrx** [comment]
Hi Martin,

I've only been able to chat with three people who have tried to test M-x find-wconfig-links... one struggled A LOT to download an .mp4 and a .vtt into the same directory without wget, the second tried to participate in one of my workshops but he couldn't install mpv on his father's laptop - he tried to, but he got stuck with a problem with permissions that he couldn't solve - and the third one got much farther, but he had never used terminals before, so he was still in the stage in which terminals are scary... but anyway, we met in person one day and he helped me to debug some things using his Windows laptop.

So: these people were "real beginners" - and they were not yet in a point in which their version of Windows would matter much, and they were also not in a point in which I could simply say "install git" or "install bash"... they would probably take many hours, or even days, just to do that...
